packages/devextreme/js/viz/series/points/bar_point.js

```diff
@@ -51,14 +51,14 @@ export default _extend({}, symbolPoint, {
         this.inVisibleArea = minX <= x + width && maxX >= x && minY <= y + height && maxY >= y;
     },
 
-    _checkVisibility: function(arg, val, minVal, rotated) {
-        const size = _abs(val - minVal);
-        const valueBeforeCorrection = Math.min(val, minVal);
+    _cacheVisibility: function(x, y, minY, rotated) {
+        y = Math.min(y, minY);
```

Comment:
Function should be pure because it lead to side effects

Reply:
I don't see any side effects it may cause